src/mustache/Context.hx

```diff
@@ -80,7 +80,13 @@ private class ContextImpl {
     }
 
     static function getField(object:Dynamic, name:String):haxe.ds.Option<Dynamic> {
-        var value = Reflect.field(object, name);
+        var value;
+        if (Std.is(object, haxe.ds.StringMap)) {
```

Comment:
`Std.downcast` will do the type check, so there's no need for `Std.is` here, simply do
```haxe
var map = Std.downcast(object, haxe.ds.StringMap);
if (map != null) {
    return map.exists(name) ? Some(map.get(name)) : None;
}
```